### PR TITLE
Soil moisture

### PR DESCRIPTION
**Pull request by CFranc22**
## Description

This is a function that pulls VSWC values from the EFI server.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by CFranc22** [comment]
I'm Christina, a student from the Harvard Forest REU program. I worked with Mike Dietze to incorporate site-specific soil moisture data into the PEcAn workflow.

**Review by mdietze — CHANGES_REQUESTED** [review]
A good start, but a few things need fixing:
* the same script seems to have been committed twice, once to the R folder and once to the scripts folder (inst). The code seems to mostly be an R function, so I think we R folder version is the one we want to retain
* Given that, the R file needs to JUST contain the function, not the homedir and site_ID that come before that. But these things might be useful for an example.
* PEcAn convention is to call the directory you're writing files to `outdir` not `homedir`
* functions need ROxygen documentation -- @helge22a can show you how RStudio ca give you a template for that
* PEcAn also asks that functions be called by namespace rather than using `library` to load whole packages. So, for example, you'd drop `library(readr)` and instead replace `read_csv` with `readr::read_csv`
* I don't know if it's coming in a separate PR from @helge22a, but there should be parallel changes to `write.configs.SIPNET` where these files are read in and used
* The latter is a high-enough level new feature that the overall PEcAn documentation should be updated to indicate that this new feature exists

**Comment by helge22a** [comment]
@mdietze I will make the changes you've requested today. The additions to the write.configs.SIPNET are apart of the larger pull request I am doing that will include all of the IC functions.

**Comment by helge22a** [comment]
@mdietze How should I go about making changes to the overall documentation?

**Comment by mdietze** [comment]
Documentation is in the `book_source` folder and its file organization mirrors the order of the compiled documentation

File: modules/data.land/inst/Pulling_VSWC.R [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Script pulling soil moisture data from target files to create a netCDF file at the end
+## vswc = mass_fraction_of_unfrozen_water_in_soil_moisture
+
+# loading necessary packages
+library(readr)
+library(dplyr)
+library(ncdf4)
+
+# set function inputs
+homedir <- "/projectnb/dietzelab/cfranci1"
```
Comment (serbinsh) [inline modules/data.land/inst/Pulling_VSWC.R:10]: But we will want to pull these hard-coded paths right?  so they work on any machine

**Pull request closed without merging**